### ISSUE TITLE
Unihertz Jelly Star 解锁 Bootloader 和刷入 Magisk 获取 Root 权限并安装相关模块

### ISSUE THREAD

**Issue by epi618** [epi618/epi618.github.io#5]
https://314729.xyz/posts/unihertz_jelly_star_解锁bootloader和刷入magisk获取root权限并安装相关模块/?code=c4b7843550165660b0ec

**Comment by scofieldpeng** [comment]
hello,看了你开箱的文章,drive上的包看起来是海外版本的固件?

**Comment by epi618** [comment]
> @scofieldpeng
> hello,看了你开箱的文章,drive上的包看起来是海外版本的固件?

是的带Google Play的固件

**Comment by scofieldpeng** [comment]
> @epi618
> > @scofieldpeng
> > hello,看了你开箱的文章,drive上的包看起来是海外版本的固件?
> 
> 是的带Google Play的固件

感谢, 国行通过其他骚操作获取到了boot.img进行root了,不得不说,官方的系统还是那么渣,几个小时就发现了好几个bug==

**Comment by epi618** [comment]
> @scofieldpeng
> > @epi618
> > > @scofieldpeng
> > > hello,看了你开箱的文章,drive上的包看起来是海外版本的固件?
> > 
> > 是的带Google Play的固件
> 
> 感谢, 国行通过其他骚操作获取到了boot.img进行root了,不得不说,官方的系统还是那么渣,几个小时就发现了好几个bug==

感觉这个公司硬件还比较新奇 但是内核不开源 自己又不好好维护系统 其实违背了开源协议

**Comment by scofieldpeng** [comment]
> @epi618
> > @scofieldpeng
> > > @epi618
> > > > @scofieldpeng
> > > > hello,看了你开箱的文章,drive上的包看起来是海外版本的固件?
> > > 
> > > 是的带Google Play的固件
> > 
> > 感谢, 国行通过其他骚操作获取到了boot.img进行root了,不得不说,官方的系统还是那么渣,几个小时就发现了好几个bug==
> 
> 感觉这个公司硬件还比较新奇 但是内核不开源 自己又不好好维护系统 其实违背了开源协议

是的,之前他们家出的pocket和slim都准备入手的,看了下重量和厚度,放弃了,他们家的jelly2我当主力机用了2年多,勉强满足需求吧,就是硬件太渣,屏幕不知道是找的什么陈年老货,一年多就黄了,新到的这个,我看也有一点黄,想了下忍了,jelly star怎么说呢,g99这个低端soc勉强跟得上时代吧,好歹还是6nm,6+128都够用了,8+256已经完全过剩, 官方系统小bug不说了,这一代居然吧支付宝和微信的指纹支付给干没了,也不知道后续会不会跟进, 暂时root把我用到的那几个app装上先凑合,看看后面要不要gsi安装Los看看

**Comment by epi618** [comment]
> @scofieldpeng
> > @epi618
> > > @scofieldpeng
> > > > @epi618
> > > > > @scofieldpeng
> > > > > hello,看了你开箱的文章,drive上的包看起来是海外版本的固件?
> > > > 
> > > > 是的带Google Play的固件
> > > 
> > > 感谢, 国行通过其他骚操作获取到了boot.img进行root了,不得不说,官方的系统还是那么渣,几个小时就发现了好几个bug==
> > 
> > 感觉这个公司硬件还比较新奇 但是内核不开源 自己又不好好维护系统 其实违背了开源协议
> 
> 是的,之前他们家出的pocket和slim都准备入手的,看了下重量和厚度,放弃了,他们家的jelly2我当主力机用了2年多,勉强满足需求吧,就是硬件太渣,屏幕不知道是找的什么陈年老货,一年多就黄了,新到的这个,我看也有一点黄,想了下忍了,jelly star怎么说呢,g99这个低端soc勉强跟得上时代吧,好歹还是6nm,6+128都够用了,8+256已经完全过剩, 官方系统小bug不说了,这一代居然吧支付宝和微信的指纹支付给干没了,也不知道后续会不会跟进, 暂时root把我用到的那几个app装上先凑合,看看后面要不要gsi安装Los看看

是的了 祝玩机开心 我已经卖掉了jelly star了 😄

**Comment by scofieldpeng** [comment]
> @epi618
> > @scofieldpeng
> > > @epi618
> > > > @scofieldpeng
> > > > > @epi618
> > > > > > @scofieldpeng
> > > > > > hello,看了你开箱的文章,drive上的包看起来是海外版本的固件?
> > > > > 
> > > > > 是的带Google Play的固件
> > > > 
> > > > 感谢, 国行通过其他骚操作获取到了boot.img进行root了,不得不说,官方的系统还是那么渣,几个小时就发现了好几个bug==
> > > 
> > > 感觉这个公司硬件还比较新奇 但是内核不开源 自己又不好好维护系统 其实违背了开源协议
> > 
> > 是的,之前他们家出的pocket和slim都准备入手的,看了下重量和厚度,放弃了,他们家的jelly2我当主力机用了2年多,勉强满足需求吧,就是硬件太渣,屏幕不知道是找的什么陈年老货,一年多就黄了,新到的这个,我看也有一点黄,想了下忍了,jelly star怎么说呢,g99这个低端soc勉强跟得上时代吧,好歹还是6nm,6+128都够用了,8+256已经完全过剩, 官方系统小bug不说了,这一代居然吧支付宝和微信的指纹支付给干没了,也不知道后续会不会跟进, 暂时root把我用到的那几个app装上先凑合,看看后面要不要gsi安装Los看看
> 
> 是的了 祝玩机开心 我已经卖掉了jelly star了 😄

哈哈,对我来说只是一个便携的通讯+支付设备,也不玩游戏啥的,比较适合我的需求吧. 现在动辄200g的“轻薄”手机还一堆广告实在受不了
